### ISSUE TITLE
How to translate some text, like "Next", "Previous", "Lesson", etc?

### ISSUE THREAD

**Issue by misibacsi** [qgis/QGIS-Documentation#8262]
## Description

I am already translating the Qgis documentation on Transifex platform. 
However, some text are not accessible to translate.
This means the end result is a mixed-language text.

Expected:
All usable text should be in the target language.

Actual behavior:
The following are examples of the not accessible texts on Transifex.
(Or I do not know where they are stored)

Lesson
Follow Along
Try Yourself
Try Yourself 1
Try Yourself 2...
In Conclusion
What’s Next?
Next
Previous

These texts are used on many pages, which otherwise are translated (or can be).

Page URL:
https://docs.qgis.org/3.28/hu/docs/training_manual/basic_map/mapviewnavigation.html
